odxtools/statetransitionref.py

```diff
@@ -108,13 +108,13 @@ def _check_applies(ref: Union["StateTransitionRef",
         return False
     elif not isinstance(
             param,
-            CodedConstParameter | PhysicalConstantParameter | TableKeyParameter | ValueParameter):
```

Comment:
weird: when I was doing #406, ruff changed this from a tuple to the union type. do you have any idea why (and also why it does not complain now)

Reply:
I added it to the ignore list in pyproject.toml.[ That rule is deprecated](https://docs.astral.sh/ruff/rules/non-pep604-isinstance/).

Reply:
cool; but could you move that into a separate PR?